addons/l10n_si/data/template/account.tax.group-si.csv

```diff
@@ -3,3 +3,7 @@
 "tax_group_95","9,5% VAT","base.si","gd_acc_260800","gd_acc_160800","9,5% DDV"
 "tax_group_5","5% VAT","base.si","gd_acc_260800","gd_acc_160800","5% DDV"
 "tax_group_0","0% VAT","base.si","gd_acc_260800","gd_acc_160800","0% DDV"
+"tax_group_8","8% VAT","base.si","gd_acc_260800","gd_acc_160800","8% DDV"
+"tax_group_so_22","22% SO VAT","base.si","gd_acc_260800","gd_acc_160800","22% SO DDV"
+"tax_group_so_95","9,5% SO VAT","base.si","gd_acc_260800","gd_acc_160800","9,5% SO DDV"
+"tax_group_so_05","5% SO VAT","base.si","gd_acc_260800","gd_acc_160800","5% SO DDV"
```

Comment:
Please put the tax ids in order, (move the `tax_group_8` over tax_group_5`) since we're changing everything I don't think the diff size matters.

Rename one of the two:
- `tax_group_so_05` -> `tax_group_so_5`
- `tax_group_5` -> `tax_group_05`

What are these SO groups btw? Samo ocenjevanje? Self-assessment, self taxation... sounds like it's Reverse Charge. In that case please follow the [Tax Naming Conventions](https://www.odoo.com/odoo/knowledge.article/1518?cids=1), it should be "RC" then.

![image](https://github.com/odoo/odoo/assets/1665365/0c964807-7d91-4aaf-bb58-3e67bcf7bb24)